core/chains/evm/config/toml/defaults.go

```diff
@@ -65,6 +73,59 @@ func init() {
 	slices.SortFunc(DefaultIDs, func(a, b *big.Big) int {
 		return a.Cmp(b)
 	})
+
+	// read the custom defaults overrides
+	dir := env.CustomDefaults.Get()
+	if dir == "" {
+		// short-circuit; no default overrides provided
+		return
+	}
+
+	// Read directory contents
+	entries, err := os.ReadDir(dir)
+	if err != nil {
+		log.Printf("error reading custom defaults override directory: %v", err)
+		return
```

Comment:
Why Print rather than fatal?
```suggestion
		log.Fatalf("error reading custom defaults override directory: %v", err)
```

Reply:
I didn't want to crash the node on boot

Reply:
Crashing is the desired behavior in the case of invalid configuration, otherwise it often goes unnoticed, especially when a traditional logger.Logger is not available.